### ISSUE TITLE
Hide/unhide the app with single command and skhd

### ISSUE THREAD

**Issue by xbladesub** [koekeishiya/yabai#1597]
Is it possible to create a command/tool that will be used to hide/unhide-focus app. I'm using `open` with skhd, but it just open then app and repeating this doesn't do anything - I want to hide the app with same shortcut

**Comment by command-z-z** [comment]
Also,is there a shortcut key or method to close the window/app:-)

**Comment by azinsharaf** [comment]
@command-z-z  `Command-W` ?

**Comment by command-z-z** [comment]
> @command-z-z `Command-W` ?

yes, but i want to config custom key in my dotfile not use mac keybind setting.

**Comment by azinsharaf** [comment]
@xbladesub Have you found a solution to this or you just closed the Issue?

**Comment by xbladesub** [comment]
> @xbladesub Have you found a solution to this or you just closed the Issue?

I found the solution - `rcmd` app for macos
